LIMITATIONS.md

```diff
@@ -0,0 +1,69 @@
+# Security limitations
+
+This document's goal is to provide a short discussion on what security
+properties access to /dev/sev-guest has, specifically in a Cloud setting.
+
+## Initial assumptions
+
+For claims in this document, we assume that the VM firmware is vendored and
+signed by the vendor, or not signed at all. This will be the common use case.
+
+If a virtual machine monitor gives you access to provide your own launch image
+and signed IDBlock, then you have full control and responsibility over what that
+image does, and how careful it is to give workloads access to the SEV
+device. Since this library is for wrapping the Linux driver /dev/sev-guest which
+will be booted from a UEFI firmware, further discussion of other uses of SEV-SNP
+are out of scope.
+
+## Implications of a vendored firmware
+
+Your Cloud service provider (CSP) may launch SEV-SNP VMs with their own build of
+UEFI firmware. This build and its signature in IDBlock are probably widely
+deployed across the CSP's fleet.  Because the firmware and IDBlock are the same
+everywhere, the measurement in the attestation report will be the same
+everywhere. The IDBlock, if provided, will also be the same everywhere.
```

Comment:
Can make the change below? Let me know if you don't agree with this change.

"The IDBlock, if provided, will also be the same everywhere."
->
"The IDBlock, if provided, may be the same everywhere."

Reply:
Done.

Reply:
Hmm.. I don't see the updated PR. Did you re-upload it?

Reply:
Ah, I amended but forgot to push. Done.